snippets/markdown/audio/tflite/Android.md

```diff
@@ -0,0 +1,26 @@
+You can test your TensorFlow Lite sound classification model on Android by following these steps:
+1. Download the [sample app](https://github.com/tensorflow/examples/tree/master/lite/examples/sound_classification/android) from GitHub.
+2. Extract the ZIP archive that you download from Teachable Machine.
+3. Copy the `soundclassifier.tflite` and `labels.txt` files from the archive to the `src/main/assets` folder in the sample app, replacing the demo model there.
+
+If you want to integrate the model into your existing app, follow these steps:
+1. Put the `soundclassifier.tflite` and `labels.txt` files into the `assets` folder in your app.
+2. Copy the [SoundClassifier.kt](https://github.com/tensorflow/examples/blob/master/lite/examples/sound_classification/android/app/src/main/java/org/tensorflow/lite/examples/soundclassifier/SoundClassifier.kt) file to your app. This file contains the source code to use the sound classification model.
+3. Initialize a `SoundClassifier` instance from your `Activity` or `Fragment` class.
```

Comment:
It seems to me that the code such as `soundClassifier.start()` and `soundClassifier.probabilities.observe()` is already included in the sample app. So the text here has the slight risk of making the readers think that the sample app is incomplete and they need to manually add those lines of code.

So perhaps make it clear that the code already exists, and the description here is just calling them out for highlighting?

Reply:
These steps are for integrating the SoundClassifier into an arbitrary app, which is different to the sample app. I added a heading to make it clearer. WDYT?

Reply:
Looks good. I think it's clearer.